_posts/2020-08-01-Fortran-Newsletter-August-2020.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "Fortran newsletter: August 2020"
+category: newsletter
+author: Ondřej Čertík, Milan Curcic, Laurence Kedward, and Jérémie Vandenplas
+---
+
+Welcome to the August 2020 edition of the monthly Fortran newsletter.
+The newsletter comes out on the first calendar day of every month
+and details Fortran news from the previous month.
+
+<ul id="page-nav"></ul>
+
+## fortran-lang.org
+
+Work has continued on the Fortran-lang website, including a new community page and additional tutorial content:
+
+* [#120](https://github.com/fortran-lang/fortran-lang.org/pull/120): updated the
+  [Book section](https://fortran-lang.org/learn/) with a comprehensive list of
+  books about Fortran
+
+Ongoing work:
+
+* [#117](https://github.com/fortran-lang/fortran-lang.org/issues/117): Adding a
+  Benchmarks section, a new dedicated repository was created at
+  https://github.com/fortran-lang/benchmarks and many details have been
+  discussed in [issues](https://github.com/fortran-lang/benchmarks/issues) there
+
+
+
+[Let us know](https://github.com/fortran-lang/fortran-lang.org/issues) if you have any suggestions for the website and its content.
+We welcome any new contributors to the website and the tutorials page in particular - see the [contributor guide](https://github.com/fortran-lang/fortran-lang.org/blob/master/CONTRIBUTING.md) for how to get started.
+
+
+## Fortran Standard Library
+
+What's new in the Fortran Standard Library:
+
+
+Some miscellaneous fixes and improvements:
+
+
+## Fortran Package Manager
+
+What's new in fpm:
+
+
+fpm is still in very early development, and we need as much help as we can get.
+Here's how you can help today:
+
+* Try to use it. Does it work? No? Let us know! Read the [fpm packaging guide](https://github.com/fortran-lang/fpm/blob/master/PACKAGING.md) to learn how to build your package with fpm.
+* Browse the [open issues](https://github.com/fortran-lang/fpm/issues) and see if you can help implement any fixes or features.
+* Adapt your Fortran package for fpm.
+* Improve the documentation.
+
+The short term goal of fpm is to make development and installation of Fortran packages with dependencies easier.
+Its long term goal is to build a rich and decentralized ecosystem of Fortran packages and create a healthy
+environment in which new open source Fortran projects are created and published with ease.
+
+## Events
+
+* [FortranCon 2020](https://tcevents.chem.uzh.ch/event/12) was held July 2 - 4.
+with many interesting talks.
+See the talk schedule
+[here](https://tcevents.chem.uzh.ch/event/12/timetable/#20200702.detailed).
+FIXME: is recordings available?
```

Comment:
They said they would be made available on [their SWITCHtube](https://tube.switch.ch/organizations/5), but nothing there yet.

Reply:
We will say that they are planned, since we want to get the newsletter out.